versioned_docs/version-2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-repositories.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Using OCI-Based Helm Chart Repositories
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-registries"/>
+</head>
+
+Helm v3 introduced storing Helm charts as [Open Container Initiative (OCI)](https://opencontainers.org/about/overview/) artifacts in container registries. With Rancher v2.9.0, you can add [OCI-based Helm chart repositories](https://helm.sh/docs/topics/registries/) alongside HTTP-based and Git-based repositories. This means that you can deploy apps that are stored as OCI artifacts. As of Rancher v2.9.0, this feature is experimental.
```

Comment:
I guess there's also the question of what experimental really means in this context. For feature flags, "experimental" means "not on by default" or "considered optional".

Reply:
> I guess there's also the question of what experimental really means in this context. 

Features classified as "experimental" are unsupported (which requires a prominent callout) and often serve as a "preview" of upcoming features.

> For feature flags, "experimental" means "not on by default" or "considered optional".

I don't think the context of being behind a feature flag affects the meaning of "experimental" either. Because of their unsupported nature, experimental features are "considered optional" regardless of whether they're behind a feature flag or not. The "not on by default" part is not accurate either as we've had things like RKE2/K3s provisioning being on by default in early 2.6, but it was experimental.